web/pandas/versions.json

```diff
@@ -4,13 +4,9 @@
         "version": "docs/dev"
     },
     {
-        "name": "1.4 (rc)",
+        "name": "1.4 (stable)",
         "version": "pandas-docs/version/1.4"
     },
-    {
-        "name": "1.3 (stable)",
-        "version": "docs"
```

Comment:
Small note that we should keep the entry with just the "docs" version (that's for the plain pandas.pydata.org/docs url), but with an updated version number -> https://github.com/pandas-dev/pandas/pull/45590